sql/src/main/java/io/crate/expression/reference/file/SourceLineNumberExpression.java

```diff
@@ -37,7 +37,7 @@
 
     public static Reference getReferenceForRelation(RelationName relationName) {
         return new Reference(
-            new ReferenceIdent(relationName, COLUMN_IDENT), RowGranularity.DOC, DataTypes.LONG, null
+            new ReferenceIdent(relationName, COLUMN_IDENT), RowGranularity.DOC, DataTypes.LONG, null, null
```

Comment:
@seut I also considered this myself but there are already 3 constructors in place 
and I thought that If added more, it would be more confusing than handy. 

What do you think?

Reply:
I'd prefer adding another one than changing lot of files/lines only for adding a `null`

Reply:
We could also implement a `Reference.Builder` to avoid all these ctors...

Reply:
I think introducing a `Reference.Builder` could also be a follow up - or we might want to think a bit more in general about where we want to go with these Reference structures.

We stream a lot of information for the execution plan that often isn't needed.

Reply:
agreed, I'll follow up on this with a separate PR